script/10_deployMultiFeedAdaptersWithoutRounds.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+echo "Instructing the shell to exit immediately if any command returns a non-zero exit status..."
+set -e
+echo "Done."
+
+echo "Navigating to the root directory of the project..."
+cd ../
+echo "Done."
+
+echo "Setting environment variables..."
+source .env
+echo "Done."
+
+echo "Creating $NETWORK directory inside deployment/artifacts/contracts directory..."
+if [ -z "$NETWORK" ]
+then
+      echo "NETWORK variable inside .env file is not set. Please set NETWORK environment variable."
+      exit 1
+else
+      if [ -d "deployment/artifacts/contracts/$NETWORK" ]
+      then
+            echo "Directory deployment/artifacts/contracts/$NETWORK already exists."
+      else
+            mkdir deployment/artifacts/contracts/$NETWORK
+      fi
+fi
+echo "Done."
+
+echo "Deploying and if enabled verifying L2MultiFeedAdapterWithoutRoundsPrimaryProd smart contract..."
+if [ -z "$CONTRACT_VERIFIER" ]
+then
+      forge script --rpc-url="$L2_RPC_URL" --broadcast -vvvv script/contracts/L2/L2MultiFeedAdapterWithoutRoundsPrimaryProd.s.sol:L2MultiFeedAdapterWithoutRoundsPrimaryProdScript
+else
+      if [ $CONTRACT_VERIFIER = "blockscout" ]
+      then
+            forge script --rpc-url="$L2_RPC_URL" --broadcast --verify --verifier blockscout --verifier-url $L2_VERIFIER_URL -vvvv script/contracts/L2/L2MultiFeedAdapterWithoutRoundsPrimaryProd.s.sol:L2MultiFeedAdapterWithoutRoundsPrimaryProdScript
+      fi
+      if [ $CONTRACT_VERIFIER = "etherscan" ]
+      then        
+            forge script --rpc-url="$L2_RPC_URL" --broadcast --verify --verifier etherscan --etherscan-api-key="$L2_ETHERSCAN_API_KEY" -vvvv script/contracts/L2/L2MultiFeedAdapterWithoutRoundsPrimaryProd.s.sol:L2MultiFeedAdapterWithoutRoundsPrimaryProdScript
+      fi
+fi
+echo "Done."
+
+echo "Deploying and if enabled verifying L2MultiFeedAdapterWithoutRoundsMainDemo smart contract..."
+if [ -z "$CONTRACT_VERIFIER" ]
+then
+      forge script --rpc-url="$L2_RPC_URL" --broadcast -vvvv script/contracts/L2/L2MultiFeedAdapterWithoutRoundsMainDemo.s.sol:L2MultiFeedAdapterWithoutRoundsMainDemoScript
+else
+      if [ $CONTRACT_VERIFIER = "blockscout" ]
+      then
+            forge script --rpc-url="$L2_RPC_URL" --broadcast --verify --verifier blockscout --verifier-url $L2_VERIFIER_URL -vvvv script/contracts/L2/L2MultiFeedAdapterWithoutRoundsMainDemo.s.sol:L2MultiFeedAdapterWithoutRoundsMainDemoScript
+      fi
+      if [ $CONTRACT_VERIFIER = "etherscan" ]
+      then        
+            forge script --rpc-url="$L2_RPC_URL" --broadcast --verify --verifier etherscan --etherscan-api-key="$L2_ETHERSCAN_API_KEY" -vvvv script/contracts/L2/L2MultiFeedAdapterWithoutRoundsMainDemo.s.sol:L2MultiFeedAdapterWithoutRoundsMainDemoScript
+      fi
+fi
+echo "Done."
```

Comment:
This is surely a useful improvement but if implemented, it would only apply to two files: `10_deployMultiFeedAdaptersWithoutRounds.sh` and `11_deployPriceFeedsWithoutRounds.sh`.
The current pattern is consistent across all other files and I’m unsure whether we want to deviate from the established approach.
If you think it makes sense to apply the suggested pattern to new files like above two, I’d be happy to make the necessary changes.

Reply:
I don't have any strong preferences over here but in general it'd be nice to have the code cleaner.

Reply:
I agree that refactoring these scripts would improve their organization and the code would look cleaner!

I would prefer not to have some files following one pattern and others using a different one, especially when all the files are essentially the same.
Since this refactor will impact on a quite big number of files and is separate from the topic of this PR, I have created [a new issue](https://github.com/LiskHQ/lisk-contracts/issues/209) to address it.
It would be best not to combine these two topics, as it may make the PR too big and difficult to review.

Reply:
Sounds good to me 👍